### PR TITLE
Fix die that does not print the value

### PR DESCRIPTION
**Pull request by davidglezz**
If status is an integer, that value will be used as the exit status and not printed

https://www.php.net/manual/en/function.exit.php

### CODE REVIEW

File: controllers/front/actions.php [hunk 1]
```diff
@@ -77,7 +77,7 @@ public function processRemove()
             die('0');
         }
 
-        die(1);
+        die('1');
```
Comment (PierreRambaud) [inline controllers/front/actions.php]: I confess that I don't understand why in the first condition we `die('0')` if something went wrong, then `die('0')` if the mail alert was successfully deleted, and to finish, `die(1)` :thinking:  Do you know where it's used?

Comment (davidglezz) [reply controllers/front/actions.php]: Fixed

**Comment by Robin-Fischer-PS** [comment]
Hi @davidglezz ! Thanks for the PR.
Could you please give us a scenario for testing this PR ? :)

**Comment by davidglezz** [comment]
I'm sorry, I wouldn't know.
In practice it was unlikely that something go wrong with this bug.

**Comment by matks** [comment]
@Robin-Fischer-PS the scenario here would be "test nothing is broken" 😅 

Do you agree @PierreRambaud ?

**Comment by PierreRambaud** [comment]
@matks Yup :+1:

**Comment by davidglezz** [comment]
Still with "Waiting for author" label.
Do you need me to do something?

**Comment by matks** [comment]
@davidglezz Sorry, one worklow issue on our side. Thanks for pointing it out, I fix this

**Comment by PierreRambaud** [comment]
Thanks @davidglezz

**Pull request merged**